src/Softeq.NetKit.Chat.SignalR/Sockets/ChannelSocketService.cs

```diff
@@ -116,11 +116,11 @@ public async Task<ChannelSummaryResponse> UpdateChannelAsync(UpdateChannelReques
 
             if (updatingChannel.Name != request.Name)
             {
-                await SendSystemMessageAsync(request.SaasUserId, request.ChannelId, new ChannelNameChangedLocalizationVisitor(updatedChannel), _systemMessagesConfiguration.ChannelNameChanged);
+                await SendSystemMessageAsync(request.SaasUserId, request.ChannelId, new ChannelNameChangedLocalizationVisitor(updatedChannel), RecipientType.AllExceptCallerConnectionId, _systemMessagesConfiguration.ChannelNameChanged);
             }
             else if (updatingChannel.PhotoUrl != request.PhotoUrl)
             {
-                await SendSystemMessageAsync(request.SaasUserId, request.ChannelId, new ChannelIconChangedLocalizationVisitor(updatedChannel), _systemMessagesConfiguration.ChannelIconChanged);
+                await SendSystemMessageAsync(request.SaasUserId, request.ChannelId, new ChannelIconChangedLocalizationVisitor(updatedChannel), RecipientType.AllExceptCallerConnectionId, _systemMessagesConfiguration.ChannelIconChanged);
```

Comment:
At this place we should notify all connections including current one.

Reply:
done